README.md

```diff
@@ -16,6 +16,9 @@
 ## Documentation
 Visit [the website](https://canopas.github.io/UIPilot/) for documentation and examples.
 
+## Examples
+Please have a look at [the examples](https://github.com/canopas/UIPilot/tree/main/Examples) to know more about its implementation with different use cases.
```

Comment:
-> Please have a look at [the examples](https://github.com/canopas/UIPilot/tree/main/Examples) to know more 
about different use cases of UIPilot.